components/MnemonicView.tsx

```diff
@@ -86,21 +92,26 @@ interface MnemonicViewProps extends Omit<FlexProps, 'onChange'> {
   placeholder: string
   value?: string[]
   toolTipProps?: TooltipProps
-  isReadOnly: boolean
+  isReadOnly?: boolean
   visible?: boolean
   onChange: (value: string[]) => void
   loaded?: boolean
+  error?: {
+    isInvalid: boolean
```

Comment:
suggestion: Since `isInvalid` can be derived from `errors`, would it be better to drop this and just do something like `const isInvalid = errors.some((val) => val)`?

Reply:
that separation was done to highlight input frame when input elements has no errors, for the phrase it means all cells are filled but in some of them words are incorrect
https://www.figma.com/file/fYw3aNgfS0qCS2jezGqv8P/Wallet-(New-Ui-Kit)?node-id=4623%3A19922&t=B2CAHq4fBVl83FXJ-0

Reply:
used different approach, hope its cleaner

Reply:
Got it, makes sense. Thanks for linking to Figma!